index.js

```diff
@@ -1,20 +1,476 @@
+    // if they type "enter classroom" the lockClassroom function will kick in. 
+  } else if (action === "enter" && target === "classroom") {
+    lockClassroom();
+    console.log(this.description);
+    return play();
+    // if they type "enter teachersLounge" the lockLounge function will kick in.
+  } else if (action === "enter" && target === "teachersLounge") {
+    lockLounge();
+    console.log(this.description);
+    return play();
+  } else if (action === "grab") {  
+    console.log(lookupTableOne[target].grab());
+    return play();
+ // } else if (action === "grab" && target =)  
+
+//return `You can't grab that.\n>_`;
+
+  } else if (action === "drop") {
+    if (
+      inventory.includes(
+        "globe",
+        "jacket",
+        "backpack",
+        "hallpass",
+        "folder",
+        "studentID"
+      )
+    ) {
+      console.log(
+        "Thank you for returning all items to the lost and found. You win!"
+      );
+      process.exit();
+    } else {
+      console.log(
+        "Sorry, you have not found all of the lost items. You cannot drop the items until you find them all! Please keep looking.."
+      );
+      return play();
+    }
+  } else {
+    await ask(`sorry, I don't understand ${answer}\n>_`);
+    play();
+  }
+
+  //function to check if changing between the rooms is allowed.
+
+  async function changeRooms(newRoom) {
+    //console.log(newRoom);
+    let validTransitions = roomsInSchool[currentRoom].canChangeTo;
+    if (validTransitions.includes(newRoom)) {
+      currentRoom = newRoom;
+      //console.log(this.description)
+      play();
+    } else {
+      console.log("You can't go that way\n>_");
+      play();
+    }
+  } //right now, when you enter a room it gives both the room description & the line on 446
+}
+
+/*----------Start Game---------------*/
+
+console.log(`You are standing in the schoolyard of your new school. It is the first day. 
+There is a door south of you to get inside the school, a couple trees to the east and other kids getting dropped off too.
+Your mom tells you as she drops you off, "don't forget your student ID, it's laying on the ground and the number is 12345!" 
+There is a sign on the door and something sitting on the steps. From here you can enter the hall... To enter a room, type enter and the room name(exactly as it's written....)\n To interact with an object, type use and the object's name.(it may sound weird to use some of the objects, but "use" is your best option.)\n To pick something up, type grab and the object's name.\n To examine something type examine and the object's name. To see what's in your inventory, type "inventory" & to see what's in a room, type "contents"`);
+play();
+
+
+//after working for the entire weekend on this, and having different functions work/break/work..we would love to go over our code with Bob or a TA to have a better understanding of how to structure it best, and how to fix some of our bugs so we can resubmit!
```

Comment:
I or any of Bob and the TAs would be happy to help with your project and fix some bugs! With that being said, I want you to be aware that we will be having a "free" week where there wont be a project assigned(there will be one bonus project if you're feeling ambitious) that you will be able to use to go back and fix some projects and resubmit. ALSO I think that it is very important to finish the project at hand before going back and working on old ones, you don't want things to snowball